CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Bcoin Release Notes & Changelog
 
+## v1.0.x
+
+### Wallet API changes
```

Comment:
Should this be something added now, or at release time? I think there are other changes that haven't been included in here yet.

Reply:
I think Javed suggested I add it now, mine as well keep track when a commit breaks something?

Reply:
Yeah.. the version number may end up as something other than v1.0.x. But that can be updated in a PR.

Reply:
I've updated the changelog with the rest of the changes in a new PR: https://github.com/bcoin-org/bcoin/pull/702